### PR TITLE
Add routing control

### PR DESCRIPTION
**Pull request by dtrucs**

### CODE REVIEW

File: src/components/react-json-schemaForm/GeometryField/utils.js [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+        getPathByCoordinates: async coordinates => {
+          if (name === 'mapbox') {
+            const [from, to] = coordinates;
+            const data = await fetch(
+              `https://api.mapbox.com/directions/v5/mapbox/${options.transit}/${from};${to}?geometries=geojson&overview=full&access_token=${accessToken}`,
+              {
+                method: 'GET',
+                headers: { 'Content-Type': 'application/json' },
+              },
+            ).then(response => response.json());
```
Comment (jrmi) [inline src/components/react-json-schemaForm/GeometryField/utils.js:43]:
```suggestion
            const data = await(await fetch(
              `https://api.mapbox.com/directions/v5/mapbox/${options.transit}/${from};${to}?geometries=geojson&overview=full&access_token=${accessToken}`,
              {
                method: 'GET',
                headers: { 'Content-Type': 'application/json' },
              },
            )).json();
```

??? 😇

Comment (dtrucs) [reply src/components/react-json-schemaForm/GeometryField/utils.js:43]: oh no :'(

File: src/components/react-json-schemaForm/GeometryField/MapInteraction.js [hunk 2]
```diff
@@ -0,0 +1,269 @@
...
+
+    const {
+      config: {
+        default: {
+          map: { accessToken },
+        },
+      },
+    } = settings;
+
+    const routingControl = {
```
Comment (jrmi) [inline src/components/react-json-schemaForm/GeometryField/MapInteraction.js:116]: Can you create functions to initialize this two values (othersControl) outside of the component ?

**Review by jrmi — COMMENTED** [review]
Minor refactoring if possible ?

File: src/components/react-json-schemaForm/GeometryField/MapInteraction.js (hunk 2 above)
Comment (dtrucs) [reply src/components/react-json-schemaForm/GeometryField/MapInteraction.js]: Done in the component (there are to much props dependence to be relevant outside)

**Review by jrmi — DISMISSED** [review]

**Review by jrmi — APPROVED** [review]

**Pull request merged**